ruling/index.ts

```diff
@@ -47,7 +47,7 @@ function run() {
   console.log("");
 
   const cli = new CLIEngine({
```

Comment:
In `eslint` 8 the [`CLIEngine` class has been removed](https://eslint.org/docs/user-guide/migrating-to-8.0.0#the-cliengine-class-has-been-removed), but the migration guide has details on the required changes.